voltaire_bundler/utils/eth_client_utils.py

```
@@ -180,34 +231,37 @@ async def send_rpc_request_to_eth_client_no_retry(
-                "Call to node rpc failed." +
-                str(traceback.format_exc()) +
-                str(excp)
-            )
-            await asyncio.sleep(1)  # in seconds
-        except:
-            logging.error(
-                str(traceback.format_exc())
-            )
-            await asyncio.sleep(1)  # in seconds
+                await asyncio.sleep(1)  # in seconds
+            except:
```

Flake8 (7.3.0)

[error] do not use bare 'except'

(E722)